### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec erro while importing from DBF file

### ISSUE THREAD

**Issue by Arlington1985** [jazzband/tablib#322]
I got following error while executing provided code:

**Code**
```
doctor_resource = resources.modelresource_factory(model=Doctor)()
dataset = tablib.Dataset()
dataset.headers=['full_name']
dataset.dbf=open('DOKTOR_ORIGINAL.DBF').read()
result = doctor_resource.import_data(dataset, dry_run=True)
if result.has_errors()==False:
    result = doctor_resource.import_data(dataset, dry_run=False)
    print("Data successfully imported")
else:
    print("Error: Data cannot be imported!")

```
**Log**

```
File "import.py", line 19, in <module>       
    dataset.dbf=open('DOKTOR.DBF').read()      
  File "/home/mypath/venv/lib/python3.6/codecs.py", line 321, in decode                                                           
    (result, consumed) = self._buffer_decode(data, self.errors, final)                        
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf5 in position 4: invalid start byte   
```

**Comment by WouterDX** [comment]
Pretty sure this is because of #215, and the fix in #350 , which did not update the dbf format in the same way as tsv/csv

**Comment by claudep** [comment]
Your error log show that your code is failing even before reaching tablib code. I guess you should open your dbf file in binary mode (`open('DOKTOR.DBF', mode='rb').read()`).

I just committed a documentation update about that.
